### PR TITLE
Update tgl.po

### PR DESCRIPTION
**Pull request by noobie213**
Good thing I remembered checking if there are new translations 😄 

Haven't translated others yet but most of those I didn't translate are because I think they're just better when I didn't translate them but not sure yet. Will be back to just typing them if I'm sure now.

Btw, I'm not sure to the "joysticks style" translation yet, and I'm curious about the "improved speed run" high scoring. Would it be like for targeted exact scores for every level, or just overall (which could be worse if so "in my opinion")? I wanna see it first how it works before I translate it.

New updates included:
- Translated new strings
- Improved "Multiplayer Help" translations
- Changed some old translations

### CODE REVIEW

**Review by SKPG-Tech — COMMENTED** [review]
Hey! It's nice to see you again.
Don't worry about being late on translations, no rush as always.

The translations though look great, just that I have one question. I'll comment on the line.

File: translations/tgl.po [hunk 1]
```diff
@@ -68,11 +68,11 @@ msgstr "Mga level ng komunidad"
 
 #: ../client/application/game/ui/shared_strings.cpp:23
 msgid "play count: %s"
-msgstr "Ilang beses nilaro?: %s"
+msgstr "Ilang beses naglaro?: %s"
```
Comment (SKPG-Tech) [inline translations/tgl.po:71]: Is the question mark here supposed to be before the colon?

Comment (noobie213) [reply translations/tgl.po:71]: yes

Comment (SKPG-Tech) [reply translations/tgl.po:71]: Alright, I'm merging.

**Review by SKPG-Tech — APPROVED** [review]
👍

**Pull request merged**